PyQt5-stubs/QtCore.pyi

```diff
@@ -9232,7 +9232,7 @@ QT_VERSION = ... # type: int
 QT_VERSION_STR = ... # type: str
 
 
-FuncT = typing.TypeVar('FuncT', bound=typing.Callable)  # For a correct pyqtSlot annotation
+FuncT = typing.TypeVar('FuncT', bound=typing.Callable[..., typing.Any])  # For a correct pyqtSlot annotation
```

Comment:
What is this change for?

https://docs.python.org/3/library/typing.html#typing.Callable
> A plain Callable is equivalent to Callable[..., Any]

Reply:
When strict, mypy doesn't like unparametrized generics.